rmw_opensplice_cpp/src/functions.cpp

```diff
@@ -66,11 +68,10 @@ rmw_init()
 {
   DDS::DomainParticipantFactory_var dp_factory = \
     DDS::DomainParticipantFactory::get_instance();
-  if (!dp_factory)
-  {
+  if (!dp_factory) {
```

Comment:
I thought braces went to the next line, like with `class`/`struct` declarations (see line 35)